feature/#508 remove pipeline config doc and added sequence doc to scenario config (#538)
Avaiga/taipy-doc: docs/manuals/core/concepts/index.md

Taipy Core is an application builder designed to help Python developers turn their data algorithms into an interactive
production-ready data-driven application. Taipy Core provides the necessary concepts for modeling, executing, and

> monitoring algorithms. In this documentation, such concepts are called scenarios.

Reviewer: ```suggestion
monitoring algorithms. The main Taipy concept to model an algorithm is called Scenario.
```

  that should be executed as a whole (or as subgraphs via sequences) and create a consistent algorithm. It can
  also contain a set of additional data nodes for related data but are not part of the executable graph.
- A [Sequence](sequence.md) represents a set of tasks connected through data nodes, that should be executed
  as a whole and forming a consistent algorithm. A sequence belongs to a scenario and can be thought of as a subgraph of
  the scenario's complete graph.
- A [Cycle](cycle.md) or work cycle is a time period corresponding to an iteration of a recurrent business problem.
  For instance, a company's sales forecast needs to be generated _every week_.
  A cycle is defined thanks to the **_Frequency_** of scenarios. For instance, if scenarios have a weekly
  frequency, then each cycle represents a particular week, and every scenario will be attached to a particular cycle
  (i.e. a week).
- A [Scope](scope.md) represents the _visibility_ of a data node in the graph of entities, and the level of its
  owner (Scenario, Cycle, Global).

!!! important "Definition: Config vs Entities"

>     Among the concepts described in this section, **data nodes**, **tasks**, **sequences**, and **scenarios** have

Reviewer: I would not talk abt sequence here. It is like Cycle. We don't need to mention it in this "important" message.

```suggestion
    Among the concepts described in this section, **data nodes**, **tasks**, and **scenarios** have
```

    two types of Taipy objects related to them: _configuration_ objects and _runtime_ objects.

    To differentiate them, the configuration objects are named **_configs_** (`DataNodeConfig`, `TaskConfig`,
    and `ScenarioConfig`). , The sequence concept does not have its own configuration object.
    However, sequence configuration can be done within the `ScenarioConfig`. In contrast,

>     the runtime objects (`DataNode`, `Task`, `Sequence`, and `Scenario`) are called **_entities_**.

Reviewer: ```suggestion
    and `ScenarioConfig`). In contrast, the runtime objects (`DataNode`, `Task`, and `Scenario`) are called **_entities_**.
```
